### PR TITLE
[Merged by Bors] - Upgrade openvino of compiler to version 2022.1.0

### LINKED ISSUE #584: Runtime has upgraded OpenVINO to version 2022.1.0, but compiler has not upgraded
Runtime has upgraded OpenVINO to version 2022.1.0 in Adlik v0.5.0, but compiler has not upgraded.

### PR DESCRIPTION
**Pull request by 982118809**
Closes #584.

Signed-off-by: 王佳越 10335419 <wang.jiayue@zte.com.cn>

### CODE REVIEW

File: model_compiler/src/model_compiler/compilers/saved_model_file_to_openvino_model.py [hunk 1]
```diff
@@ -14,8 +14,8 @@ def _get_optimize_params(input_model, output_dir, config):
               'saved_model_dir': input_model,
               'output_dir': output_dir}
     # if enble_nhwc_to_nchw=None or False set --disable_nhwc_to_nchw
-    if not config.enable_nhwc_to_nchw:
-        params['disable_nhwc_to_nchw'] = None
+    # if not config.enable_nhwc_to_nchw:
```
Comment (shiyingjin) [inline model_compiler/src/model_compiler/compilers/saved_model_file_to_openvino_model.py:17]: Remove the commented code

File: model_compiler/src/model_compiler/compilers/saved_model_to_openvino_model.py [hunk 2]
```diff
@@ -16,8 +16,8 @@ def _get_optimize_params(input_model, output_dir, config, inputs, outputs):
               'input': ','.join(item.name.split(':')[0] for item in inputs),
               'output': ','.join(item.name.split(':')[0] for item in outputs)}
     # if enble_nhwc_to_nchw=None or False set --disable_nhwc_to_nchw
-    if not config.enable_nhwc_to_nchw:
-        params['disable_nhwc_to_nchw'] = None
+    # if not config.enable_nhwc_to_nchw:
+    #     params['disable_nhwc_to_nchw'] = None
```
Comment (shiyingjin) [inline model_compiler/src/model_compiler/compilers/saved_model_to_openvino_model.py:20]: Remove the commented code

File: model_compiler/src/model_compiler/compilers/tf_frozen_graph_model_file_to_openvino_model.py [hunk 3]
```diff
@@ -14,8 +14,8 @@ def _get_optimize_params(input_model, output_dir, config):
               'input_model': input_model,
               'output_dir': output_dir}
     # if enble_nhwc_to_nchw=None or False set --disable_nhwc_to_nchw
-    if not config.enable_nhwc_to_nchw:
-        params['disable_nhwc_to_nchw'] = None
+    # if not config.enable_nhwc_to_nchw:
+    #     params['disable_nhwc_to_nchw'] = None
```
Comment (shiyingjin) [inline model_compiler/src/model_compiler/compilers/tf_frozen_graph_model_file_to_openvino_model.py:18]: Remove the commented code

File: model_compiler/src/model_compiler/compilers/tf_model_file_to_openvino_model.py [hunk 4]
```diff
@@ -14,8 +14,8 @@ def _get_optimize_params(input_model, output_dir, config):
               'input_meta_graph': input_model,
               'output_dir': output_dir}
     # if enble_nhwc_to_nchw=None or False set --disable_nhwc_to_nchw
-    if not config.enable_nhwc_to_nchw:
-        params['disable_nhwc_to_nchw'] = None
+    # if not config.enable_nhwc_to_nchw:
+    #     params['disable_nhwc_to_nchw'] = None
```
Comment (shiyingjin) [inline model_compiler/src/model_compiler/compilers/tf_model_file_to_openvino_model.py:18]: Remove the commented code

File: model_compiler/src/model_compiler/openvino_util.py [hunk 5]
```diff
@@ -141,8 +141,8 @@ def from_env(env: Mapping[str, str]) -> 'Config':
         temp_max_batch_size = env.get('MAX_BATCH_SIZE')
         max_batch_size = int(temp_max_batch_size) if temp_max_batch_size else None
         # if set enable_nhwc_to_nchw to a non-zero value, the optimizer will transform the model format
-        temp_enable_nhwc_to_nchw = env.get('ENABLE_NHWC_TO_NCHW')
-        enable_nhwc_to_nchw = bool(int(temp_enable_nhwc_to_nchw)) if temp_enable_nhwc_to_nchw else None
+        # temp_enable_nhwc_to_nchw = env.get('ENABLE_NHWC_TO_NCHW')
+        # enable_nhwc_to_nchw = bool(int(temp_enable_nhwc_to_nchw)) if temp_enable_nhwc_to_nchw else None
```
Comment (shiyingjin) [inline model_compiler/src/model_compiler/openvino_util.py:145]: Remove the commented code

File: model_compiler/tests/model_compiler/compilers/test_saved_model_file_to_openvino_model.py [hunk 6]
```diff
@@ -113,52 +138,52 @@ def test_compile_with_all_params_with_shape(self):
...
+    #                                    'enable_nhwc_to_nchw': True,
+    #                                    'max_batch_size': 1})
+    #         compiled = compiler.compile_source(SavedModelFile(model_path=model_dir), config)
+    #         self.assertEqual(compiled.get_inputs(),
+    #                          [ModelInput(name='x', data_type=tf.float32.as_datatype_enum,
+    #                                      format=ModelInput.FORMAT_NONE, dims=[4, 2, 3]),  # pylint: disable=no-member
+    #                           ModelInput(name='y', data_type=tf.float32.as_datatype_enum,
+    #                                      format=ModelInput.FORMAT_NONE, dims=[4, 2, 3])])  # pylint: disable=no-member
+    #         self.assertEqual(compiled.get_outputs(),
+    #                          [ModelOutput(name='z', data_type=tf.float32.as_datatype_enum, dims=[4, 2, 3])])
```
Comment (shiyingjin) [inline model_compiler/tests/model_compiler/compilers/test_saved_model_file_to_openvino_model.py:155]: Remove the commented code

File: model_compiler/tests/model_compiler/compilers/test_saved_model_to_openvino_model.py [hunk 7]
```diff
@@ -63,16 +77,16 @@ def test_compile_with_all_params_with_shape(self):
...
+    #                                'max_batch_size': 1,
+    #                                'enable_nhwc_to_nchw': True})
+    #     compiled = compiler.compile_source(source=_make_saved_model(), config=config)
+    #     self.assertEqual(compiled.get_inputs(),
+    #                      [ModelInput(name='x', data_type=tf.float32.as_datatype_enum,
+    #                                  format=ModelInput.FORMAT_NONE, dims=[4, 2, 3]),  # pylint: disable=no-member
+    #                       ModelInput(name='y', data_type=tf.float32.as_datatype_enum,
+    #                                  format=ModelInput.FORMAT_NONE, dims=[4, 2, 3])])  # pylint: disable=no-member
+    #     self.assertEqual(compiled.get_outputs(),
+    #                      [ModelOutput(name='z', data_type=tf.float32.as_datatype_enum, dims=[4, 2, 3])])
```
Comment (shiyingjin) [inline model_compiler/tests/model_compiler/compilers/test_saved_model_to_openvino_model.py:92]: Remove the commented code

File: model_compiler/tests/model_compiler/compilers/test_tf_forzen_graph_file_to_openvino_model.py [hunk 8]
```diff
@@ -121,18 +136,18 @@ def test_compile_with_all_params_with_shape(self):
...
+    #                                    'enable_nhwc_to_nchw': True,
+    #                                    'max_batch_size': 1})
+    #         compiled = compiler.compile_source(FrozenGraphFile(model_path=model_file.name), config)
+    #         self.assertEqual(compiled.get_inputs(),
+    #                          [ModelInput(name='x', data_type=tf.float32.as_datatype_enum,
+    #                                      format=ModelInput.FORMAT_NONE, dims=[4, 2, 3]),  # pylint: disable=no-member
+    #                           ModelInput(name='y', data_type=tf.float32.as_datatype_enum,
+    #                                      format=ModelInput.FORMAT_NONE, dims=[4, 2, 3])])  # pylint: disable=no-member
+    #         self.assertEqual(compiled.get_outputs(),
+    #                          [ModelOutput(name='z', data_type=tf.float32.as_datatype_enum, dims=[4, 2, 3])])
```
Comment (shiyingjin) [inline model_compiler/tests/model_compiler/compilers/test_tf_forzen_graph_file_to_openvino_model.py:153]: Remove the commented code

File: model_compiler/tests/model_compiler/compilers/test_tf_model_file_to_openvino_model.py [hunk 9]
```diff
@@ -123,19 +135,19 @@ def test_compile_with_all_params_with_shape(self):
...
+    #                                    'enable_nhwc_to_nchw': True,
+    #                                    'max_batch_size': 1})
+    #         compiled = compiler.compile_source(TfModelFile(model_path=model_path), config)
+    #         self.assertEqual(compiled.get_inputs(),
+    #                          [ModelInput(name='x', data_type=tf.float32.as_datatype_enum,
+    #                                      format=ModelInput.FORMAT_NONE, dims=[4, 2, 3]),  # pylint: disable=no-member
+    #                           ModelInput(name='y', data_type=tf.float32.as_datatype_enum,
+    #                                      format=ModelInput.FORMAT_NONE, dims=[4, 2, 3])])  # pylint: disable=no-member
+    #         self.assertEqual(compiled.get_outputs(),
+    #                          [ModelOutput(name='z', data_type=tf.float32.as_datatype_enum, dims=[4, 2, 3])])
```
Comment (shiyingjin) [inline model_compiler/tests/model_compiler/compilers/test_tf_model_file_to_openvino_model.py:153]: Remove the commented code

**Review by shiyingjin — CHANGES_REQUESTED** [review]

File: model_compiler/src/model_compiler/openvino_util.py [hunk 10]
```diff
@@ -151,7 +146,7 @@ def from_env(env: Mapping[str, str]) -> 'Config':
                       output_names=output_names,
                       data_type=data_type,
                       max_batch_size=max_batch_size,
-                      enable_nhwc_to_nchw=enable_nhwc_to_nchw,
+
```
Comment (NaNAGISaSA) [inline model_compiler/src/model_compiler/openvino_util.py:149]: remove blank line

File: model_compiler/src/model_compiler/openvino_util.py [hunk 11]
```diff
@@ -129,7 +127,7 @@ def from_json(value: Mapping[str, Any]) -> 'Config':
                       output_names=value.get('output_names'),
                       data_type=value.get('data_type'),
                       max_batch_size=value.get('max_batch_size'),
-                      enable_nhwc_to_nchw=value.get('enable_nhwc_to_nchw'),
+
```
Comment (NaNAGISaSA) [inline model_compiler/src/model_compiler/openvino_util.py:130]: remove blank line

**Review by NaNAGISaSA — CHANGES_REQUESTED** [review]

**Review by shiyingjin — APPROVED** [review]

**Review by NaNAGISaSA — APPROVED** [review]

**Pull request closed without merging**